pkg/applyconfigurations/testdata/cronjob/applyconfiguration/utils.go

```diff
@@ -0,0 +1,37 @@
+//go:build !ignore_autogenerated
+// Code generated by applyconfigurations. DO NOT EDIT.
+
+package testapplyconfiguration
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
+	testing "k8s.io/client-go/testing"
+	cronjob "sigs.k8s.io/controller-tools/pkg/applyconfigurations/testdata/cronjob"
+	internal "sigs.k8s.io/controller-tools/pkg/applyconfigurations/testdata/cronjob/testapplyconfiguration/internal"
+	testdatacronjob "sigs.k8s.io/controller-tools/pkg/applyconfigurations/testdata/cronjob/testapplyconfiguration/testdata/cronjob"
+)
+
+// ForKind returns an apply configuration type for the given GroupVersionKind, or nil if no
+// apply configuration type exists for the given GroupVersionKind.
+func ForKind(kind schema.GroupVersionKind) interface{} {
```

Comment:
Is there any way we could have some unit test that tries to use this code and does some minimal verification it "works"?